frontend/src/components/GeneralCharts.tsx

```diff
@@ -0,0 +1,85 @@
+import { IconProp } from '@fortawesome/fontawesome-svg-core'
+import dynamic from 'next/dynamic'
+import React from 'react'
+import type { ApexChartLabelSeries } from 'types/healthMetrics'
+import SecondaryCard from 'components/SecondaryCard'
+
+// Importing Chart dynamically to avoid SSR issues with ApexCharts
+const Chart = dynamic(() => import('react-apexcharts'), {
+  ssr: false,
+})
+
+const LineChart: React.FC<{
+  title: string
+  series: ApexChartLabelSeries[]
+  labels?: string[]
+  icon?: IconProp
+}> = ({ title, series, labels, icon }) => {
+  return (
+    <SecondaryCard title={title} icon={icon}>
+      <Chart
+        options={{
+          xaxis: {
+            categories: labels,
+          },
+        }}
+        series={series}
+        height={450}
+      />
+    </SecondaryCard>
+  )
+}
+
+const RadialChart: React.FC<{
+  title: string
+  icon?: IconProp
+  labels: string[]
+  series: number[]
+}> = ({ title, series, icon, labels }) => {
+  const colors = series.map((days) => {
```

Comment:
I saw your comment in today's update about colors for dark theme - I'm pretty sure we'll be able to pass in values dynamically based on the theme.
We could get the theme with `import { useTheme } from 'next-themes'` and then:
```suggestion
  const { theme } = useTheme();
  const colors = series.map((days) => {
```

And then render different colors based on that value.

Reply:
I did however plan to update requirements for this last chart, but you're so fast to implement stuff 😅

There should actually be just 1 breakpoint in this chart where before it - it's good, green.. and after - it's red and bad'. But since the path from green to red goes through orange - we'll still have it there. The numbers for that breakpoint are different based on the project level.
For example (totally random numbers here!), for Flagship level project there should be at least 1 release a month. So, if they are pass 30 days since last release - it should show days number and the chart at the red. 

Does that make sense? I'm not sure if I was able to explain it well. 
Let me know in Slack if you need me to clarify this. I can try to draw a picture :)